cri/v1alpha2/cri.go

```diff
@@ -306,13 +326,6 @@ func (c *CriManager) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 		}()
 	}
 
-	sandboxMeta := &SandboxMeta{
-		ID:        id,
-		Config:    config,
-		NetNSPath: netnsPath,
-		Runtime:   config.Annotations[anno.KubernetesRuntime],
-	}
-
 	if _, ok := config.Annotations[anno.LxcfsEnabled]; ok {
```

Comment:
could we put it before create sandbox? I think the validation should be checked at first